e2e/autoscript-suites/src/suites/authz-txn-basic-redirect.test.ts

```diff
@@ -1,7 +1,7 @@
 /*
  * @forgerock/javascript-sdk
  *
- * authz-txn-basic.test.ts
+ * authz-txn-basic-redirect.test.ts
```

Comment:
I decided to split out the authz-txn-basic test into two - one handling advice as redirect, the other as JSON